account_payment_order_return/wizards/account_payment_line_create.py

```diff
@@ -2,18 +2,31 @@
 # Copyright 2021 Tecnativa - João Marques
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class AccountPaymentLineCreate(models.TransientModel):
     _inherit = "account.payment.line.create"
 
     include_returned = fields.Boolean(string="Include move lines from returns")
 
-    def _prepare_move_line_domain(self):
-        domain = super()._prepare_move_line_domain()
+    @api.depends(
+        "date_type",
+        "move_date",
+        "due_date",
+        "journal_ids",
+        "invoice",
+        "target_move",
+        "allow_blocked",
+        "payment_mode",
+        "partner_ids",
+    )
+    def _compute_move_line_domain(self):
+        result = super()._compute_move_line_domain()
+        domain = self.move_line_domain
         if not self.include_returned:
             domain += [
                 ("move_id.returned_payment", "=", False),
             ]
```

Comment:
no habrá que iterar? `for line in self` ??

Reply:
no, es un wizard y estoy heredando este método que ya tiene un ensure_one()
 https://github.com/aurestic/bank-payment/blob/1371268272aef8a722e9ab695379c6b846c6cfa1/account_payment_order/wizard/account_payment_line_create.py#L81